test/phpunit/FactureTest.php

```diff
@@ -56,20 +56,9 @@ class FactureTest extends CommonClassTest
 	 */
 	public static function setUpBeforeClass(): void
 	{
-		global $conf,$user,$langs,$db;
-
-		if (!isModEnabled('facture')) {
-			print __METHOD__." module customer invoice must be enabled.\n";
-			die(1);
-		}
-		if (isModEnabled('ecotaxdeee')) {
-			print __METHOD__." ecotaxdeee module must not be enabled.\n";
-			die(1);
-		}
-
```

Comment:
Why removing the begin of transaction. We want to have nothing modified in database when test is launched manually for 1 file only.

Reply:
`$db->begin()` is already in the parent class `CommonClassTest`.